### DISCUSSION TITLE
How to download a file here?

### DISCUSSION THREAD

**Discussion by apaclin** [community/community#45072] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, I'm trying to download a program "BetterDummy", but don't know how to do it. When you press "go to file" there's a list of many files and when you press "code -> download zip" it downloads a readme file that doesn't even open. So how to do it?

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @apaclin! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by apaclin** [comment]
It doesn't work.[image: Screenshot 2023-01-27 at 16.37.29.png]

Best,
Alex Paclin.
Musician, Harmonica Player, Singer.
facebook.com/apaclinmusic


вт, 24 янв. 2023 г. в 07:11, Jermyiah ***@***.***>:

**Reply by jge162** [reply]
Is it downloading a zip file?

**Comment by apaclin** [comment]
Wait, it's a command, where do I run it?

Best,
Alex Paclin.
Musician, Harmonica Player, Singer.
facebook.com/apaclinmusic


пт, 27 янв. 2023 г. в 16:37, Alex Paclin ***@***.***>:

**Reply by jge162** [reply]
No command is needed. Just click code. Then download zip.

**Reply by jge162** [reply]
You are using a computer to download the file correct?

**Reply by jge162** [reply]
Just double click on zip folder. And it will open up the folder containing source code.

**Comment by apaclin** [comment]
It doesn't work, yes, I'm using the computer.

Best,
Alex Paclin.
Musician, Harmonica Player, Singer.
facebook.com/apaclinmusic


пт, 27 янв. 2023 г. в 16:46, Jermyiah ***@***.***>:

**Reply by jge162** [reply]
Post a screenshot of error here please.

**Comment by apaclin** [comment]
Already did.

Best,
Alex Paclin.
Musician, Harmonica Player, Singer.
facebook.com/apaclinmusic


пт, 27 янв. 2023 г. в 16:58, Jermyiah ***@***.***>:

**Reply by jge162** [reply]
Drag and drop it into the chat box. You can’t send it via email

**Comment by jge162** [comment]
[image: Screenshot 2023-01-27 at 16.37.29.png]

**Reply by jge162** [reply]
That’s not a screenshot.

**Comment by apaclin** [comment]
Best,
Alex Paclin.
Musician, Harmonica Player, Singer.
facebook.com/apaclinmusic


пт, 27 янв. 2023 г. в 17:01, Jermyiah ***@***.***>:

**Reply by jge162** [reply]
You need to log into GitHub and drop the image into the chat box. You cannot reply email it.

**Reply by jge162** [reply]
Once you log in and post a picture we can go from there.

**Comment by apaclin** [comment]
https://github.com/topics/chatbox
Here? How?

Best,
Alex Paclin.
Musician, Harmonica Player, Singer.
facebook.com/apaclinmusic


пт, 27 янв. 2023 г. в 17:04, Jermyiah ***@***.***>:

**Reply by jge162** [reply]
Sorry won’t be able to help you.  Best of luck.
